apps/backend/src/clickhouse-o11y/clickhouse-o11y.service.ts

```diff
@@ -0,0 +1,79 @@
+import { createClient, type ClickHouseClient } from '@clickhouse/client';
+import { Logger, Metrics } from '@logdash/js-sdk';
+import { Injectable } from '@nestjs/common';
+import { getEnvConfig } from 'src/shared/configs/env-configs';
+
+enum PerformanceMetric {
```

Comment:
Please somehow prefix these metrics with `clickhouse` because right now it looks as if these metrics were related to the node process.